training/model_management/src/azureml/model/mgmt/processors/transformers/config.py

```diff
@@ -22,7 +22,7 @@
 
 # HF flavor patterns
 MODEL_CONFIG_FILE_PATTERN = r"^config\.json$"
-MODEL_FILE_PATTERN = r"^pytorch.*\.bin$"
+MODEL_FILE_PATTERN = r"^pytorch.*$"
```

Comment:
Does this cover all files that we support?

Reply:
it covers model bin files and json file that captures details. I am adding to include all `.py` files as well, which might be needed for loading up model.  Adding in PR: https://github.com/Azure/azureml-assets/pull/660